jsapp/js/components/submissions/mediaCell.es6

```diff
@@ -56,25 +95,43 @@ class MediaCell extends React.Component {
             // Give the user a way to see the full file name
             title={mediaName}
           >
-            {truncatedFileName}
+            {titleText}
           </bem.TableMediaPreviewHeader__label>
         </bem.TableMediaPreviewHeader__title>
 
-        {/*TODO this doesn't start a `save as` but instead opens media in tab*/}
-        <a
-          className='kobo-light-button kobo-light-button--blue'
-          href={mediaURL}
-          download=''
-        >
-          {t('download')}
-          <i className='k-icon k-icon-download'/>
-        </a>
+        <bem.TableMediaPreviewHeader__options>
+          {this.props.mediaURL &&
+            <a
+              className='kobo-light-button kobo-light-button--blue'
+              // TODO: once we get this button to `save as`, remove this target
+              target='_blank'
+              href={mediaURL}
+            >
+              {t('download')}
+
+              <i className='k-icon k-icon-download'/>
+            </a>
+          }
+
+          <a
+            className='kobo-light-button kobo-light-button--gray'
+            // TODO: point this to submission processing modal
+            href={'#'}
+          >
+            {t('process')}
+
+            <i className='k-icon k-icon-arrow-up'
+              //TODO: Need to add icon for top-right pointing arrow
```

Comment:
I think we should comment out the button for now, as we are merging to `beta` - or switch base branch to `feature/nlp`

Reply:
BTW. ~~do you need the icon right now?~~ I've just merged the icon you need to `beta` :) Try `k-icon-arrow-up-right`

Reply:
Great point, I'll comment it out for now but still use the correct icon :D